### PR TITLE
Log deleted models + read stats + try to fix pull request __Update issue

### PR DESCRIPTION
**Pull request by tovbinm**
## Description

Log deleted models + read stats + try to fix pull request __Update issue

## Type of change
- [x] Bug fix
- [x] New feature
- [ ] Breaking change

### CODE REVIEW

File: destinations/faros-destination/src/converters/github/pull_request_stats.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export class GithubPullRequestStats extends Converter {
       {
         model: 'vcs_PullRequest__Update',
         record: {
+          at: record.record.emitted_at,
```
Comment (tovbinm) [inline destinations/faros-destination/src/converters/github/pull_request_stats.ts:25]: @eskrm @cjwooo I tried adding `at` to all `__Update` records, by no luck. How else can I make sure the updates come after the upserts when the revision closes?

Comment (cjwooo) [reply destinations/faros-destination/src/converters/github/pull_request_stats.ts:25]: The Upsert records might also need to have an `at` value.

Comment (tovbinm) [reply destinations/faros-destination/src/converters/github/pull_request_stats.ts:25]: hmm, and there I can probably just put `0`?

Comment (tovbinm) [reply destinations/faros-destination/src/converters/github/pull_request_stats.ts:25]: Works!!!

**Review by cjwooo — APPROVED** [review]

**Pull request merged**